core/include/usModuleImport.h

```diff
@@ -51,15 +51,7 @@ US_END_NAMESPACE
  * \sa \ref MicroServices_StaticModules
  */
 #define US_INITIALIZE_STATIC_MODULE(_module_name)                          \
-  extern "C" void _us_import_module_initializer_ ## _module_name();        \
-  struct StaticModuleInitializer_ ## _module_name                          \
-  {                                                                        \
-    StaticModuleInitializer_ ## _module_name()                             \
-    {                                                                      \
-      _us_import_module_initializer_ ## _module_name();                    \
-    }                                                                      \
-  };                                                                       \
-  static StaticModuleInitializer_ ## _module_name _InitializeModule_ ## _module_name;
+  extern "C" void _us_import_module_initializer_ ## _module_name();
```

Comment:
I built a static CppMicroServices build on Windows 64-bit and ran the unit tests. Aside from one failure in usModuleTest.cpp (which was resolved by removing the ifdef on line 116) and the already broken usLogTest unit tests, all other unit tests passed.

I will make sure that I test static CppMicroServices builds as I continue to make changes.